WordPress/gutenberg
# DataViews Pages: Dynamic Frame title as per active view for Pages and Templates editor pages

## What?


In DataList views for pages, add a subtitle to display the activeView.

## Why?


Closes: #56199 

This requirement is first mentioned in the above issue.

But a more refined requirement can be found in this comment - https://github.com/WordPress/gutenberg/pull/56241#issuecomment-1816762258

Just looking at the whitespace of the DataList Views of pages, it might be a little unclear what activeView you are on. That's why we can add a sub-title below the title "Pages" to display the activeView over there.

## How?


This PR adds a sub-title for DataViews screen for pages, which displays the activeView.

## Testing Instructions





Please see the below screencast section or refer the steps below.

- Open the site-editor.
- Go to "Pages".
- Now switch the active view by clicking on the left sidebar.
- As you change the views you will see that the subtitle under the title "Pages" also changes accordingly.

### Testing Instructions for Keyboard


N/A

## Screenshots or screencast 


https://github.com/WordPress/gutenberg/assets/39427067/1019c127-dcf1-42db-9b99-0159163b54b6

**OLD VIDEO:**

https://github.com/WordPress/gutenberg/assets/39427067/94712b02-8a93-4527-a957-7c93f2816073
